Open_Source/6.Popular_Projects/6.3_PIP.md

```diff
@@ -0,0 +1,242 @@
+<p align="center">
+  <img src="https://files.realpython.com/media/What-is-PIP_Watermarked.c46f49dc33f9.jpg" />
```

Comment:
Replace this image with another image or cropped that, coz there I see `Real Python logo` which we don't want in our `docs`

Reply:
Hey !! Could you review it again. I have removed the watermark.